packages/dcos-diagnostics/buildinfo.json

```diff
@@ -2,11 +2,11 @@
   "requires": ["exhibitor"],
   "single_source" : {
     "kind": "git",
-    "git": "https://github.com/dcos/3dt.git",
-    "ref": "6b560880d5a2419b8bad387d06d9355d3ad207ab",
+    "git": "https://github.com/dcos/dcos-diagnostics.git",
+    "ref": "ec7fb9d99fcb77120978f7dfbe382aca7702102d",
     "ref_origin": "master"
   },
-  "username": "dcos_3dt",
+  "username": "dcos_diagnostics",
```

Comment:
the bootstrap will create new directories and generate new service accounts, so we should be all good. I do not see any files that 3dt might own. Maybe diagnostics bundles.. The only reason why customer would care about bundles is when we asked them to provide it for us. So it should not be a problem.